### DISCUSSION TITLE
Simulate program with Zone included (Blend radius)

### DISCUSSION THREAD

**Discussion by Braghebrevi** [visose/Robots#170] [Q&A] [answered]
First of all, amazing plugin. Smooth, fast and with an exceptionally rapid learning curve.
That said, a question: how can a program simulation made to include the approximation zone? I would expect it to be the default behaviour, but it is not, and even after inserting both global and per-target Zone values in the Create Target component, I see no difference in the path described by the tcp during the simulation. I am simulating the movement of a UR10 robot, the KULeuven-UR10 preset in the robots library. Thanks in advance!

**Comment by visose** [comment, marked as answer]
Hi Braghebrevi, thanks.
Unfortunately, the plugin never supported simulating approximation zones, it's one of the oldest items in the todo list. Although it will never simulate the exact trajectory, something that is close could be added. But it's better to not rely on the precise trajectory around the approx zones.
